src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/CertificateExtensionsCommon.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Internal.Cryptography.Pal
+{
+    internal static class CertificateExtensionsCommon
```

Comment:
I'm not seeing what value this set of generic methods provides; particularly since it fans back in to a switch on the generic type parameter...